### PR TITLE
Create footer bar and indicate document name if exists

### LINKED ISSUE #205: Show footer bar indicating filename and if file contains changes
Like in desktop apps

### PR DESCRIPTION
**Pull request by SkayDks**
Closed #205

### CODE REVIEW

File: src/pods/footer/footer.component.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import classes from './footer.pod.module.css';
+import { useCanvasViewSettingsContext } from '@/core/providers';
+
+export const FooterComponent: React.FC = () => {
+  const { filename } = useCanvasViewSettingsContext();
+
+  const isNewDocument = () =>
+    filename ? `File name: ${filename}` : 'New document';
+
+  const documentName = isNewDocument();
```
Comment (antonio06) [inline src/pods/footer/footer.component.tsx:11]: remove documentName

File: src/pods/footer/footer.component.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  return (
+    <>
+      <span className={classes.footerText}>{documentName}</span>
```
Comment (antonio06) [inline src/pods/footer/footer.component.tsx:15]: use here isNewDocument  directly

File: src/pods/footer/footer.component.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import classes from './footer.pod.module.css';
+import { useCanvasViewSettingsContext } from '@/core/providers';
+
+export const FooterComponent: React.FC = () => {
+  const { filename } = useCanvasViewSettingsContext();
+
+  const isNewDocument = () =>
+    filename ? `File name: ${filename}` : 'New document';
```
Comment (antonio06) [inline src/pods/footer/footer.component.tsx:9]: Remove File name, only ${filename} or New document

File: src/pods/footer/footer.component.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import classes from './footer.pod.module.css';
+import { useCanvasViewSettingsContext } from '@/core/providers';
+
+export const FooterComponent: React.FC = () => {
+  const { filename } = useCanvasViewSettingsContext();
+
+  const isNewDocument = () => (filename ? filename : 'New document');
```
Comment (brauliodiez) [inline src/pods/footer/footer.component.tsx:8]: Let's avoid magic strings on the middle of the code, create a const on line 4 and add something like:

```ts
const NEW_DOCUMENT_NAME = "New Document";
```

When you use _is...._ you assume it's a boolean (true/false), let's avoid this in order to get the code more readable:

```diff
- const isNewDocument = () => (filename ? filename : 'New document');
- const documentName = () => (filename ? filename : NEW_DOCUMENT_NAME);
```

**Pull request merged**